classes/class-wc-connect-shipping-label.php

```diff
@@ -10,12 +10,13 @@ public function __construct( WC_Connect_Service_Settings_Store $settings_store )
 			$this->settings_store = $settings_store;
 		}
 
-		protected function get_form_data( WC_Order $order ) {
-			$form_data = array();
-			$contents = array();
-
+		protected function get_individual_packages( $order ) {
```

Comment:
Please rename this function as `get_items_as_individual_packages` to make it clearer what it does